gmshooter-v2/.specify/scripts/bash/setup-plan.sh

```
@@ -0,0 +1,61 @@
+        *) 
+            ARGS+=("$arg") 
+            ;;
+    esac
+done
+
+# Get script directory and load common functions
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+source "$SCRIPT_DIR/common.sh"
+
+# Get all paths and variables from common functions
+eval $(get_feature_paths)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)